src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -247,7 +249,6 @@ private static ProcessHelper CreateProcHelper(string command, string args, strin
                 ProcessWillExit = true,
                 Executable = command,
                 Arguments = args,
-                WorkingDirectory = workingDir,
```

Comment:
Removing setting the working directory here because the setup process cannot delete the folder that's its working directory.